ambersim/rl/pendulum/swingup.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
+from typing import Any, Dict, Optional, Union
+
+import jax
+import jax.numpy as jnp
+import mujoco as mj
+from flax import struct
+from mujoco import mjx
+
+from ambersim.rl.base import MjxEnv, State
+from ambersim.utils.io_utils import load_mj_model_from_file
+
+
+@struct.dataclass
+class PendulumSwingupConfig:
+    """Config dataclass for pendulum swingup."""
+
+    # model path: scene.xml contains ground + other niceties in addition to the pendulum
+    model_path: Union[Path, str] = "models/pendulum/scene.xml"
+
+    # number of "simulation steps" for every control input
+    # in pendulum.xml, we use a time step of 0.001, which is the "framerate of reality"
```

Comment:
these comments are actually outdated, we should fix it after things stabilize in the branch